backend/endpoints/responses/oauth.py

```diff
@@ -1,4 +1,6 @@
-from typing_extensions import NotRequired, TypedDict
+from typing import NotRequired
+
+from typing_extensions import TypedDict
```

Comment:
Getting formatter errors for these changes

```
 5:1   high  Module "typing" has no attribute "NotRequired"                                        mypy/attr-defined  
 5:1   note  Use `from typing_extensions import NotRequired` instead. See                          mypy/note          
               https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-new-additions-to-t                     
               he-typing-module
```

Reply:
The formatter should be running with an older Python version, as `NotRequired` was added to `typing` [in Python 3.11](https://docs.python.org/3/library/typing.html#typing.NotRequired).

Reply:
Last commit updates Python version used by trunk, that should fix the CI run!